packages/botfuel-dialog/src/dialogs/prompt-dialog.js

```diff
@@ -68,13 +68,13 @@ class PromptDialog extends Dialog {
    * remainingCandidates (candidates minus candidates used) and
    * newValue (value we matched with the parameter)
    */
-  matchParameterWithCandidates(parameter, candidates = [], initialValue) {
-    const sameDimCandidates = candidates.filter(candidate => candidate.dim === parameter.dim);
+  matchParameterWithCandidates(entity, candidates = [], initialValue) {
```

Comment:
I would have rename this method in `matchEntityWithCandidates` because `parameter` has been renamed in `entity`